src/backend/base/langflow/components/processing/structured_output.py

```
@@ -96,6 +91,14 @@ class StructuredOutputComponent(Component):
                     "options": ["str", "int", "float", "bool", "dict"],
                     "default": "str",
                 },
+                {
+                    "name": "multiple",
+                    "display_name": "As List",
+                    "type": "boolean",
+                    "description": "Set to True if this output field should be a list of the specified type.",
```

Pylint (3.3.7)

[convention] Line too long (110/100)

(C0301)